pkgs/development/libraries/libnitrokey/default.nix

```diff
@@ -11,12 +11,19 @@ stdenv.mkDerivation (finalAttrs: {
   pname = "libnitrokey";
   version = "3.8";
 
-  src = fetchFromGitHub {
+  src = fetchFromGitHub ({
     owner = "Nitrokey";
     repo = "libnitrokey";
     rev = "v${finalAttrs.version}";
+  } // (if stdenv.isDarwin then {
+    # On OSX, libnitrokey depends on a custom version of hidapi in a submodule.
+    # Monitor https://github.com/Nitrokey/libnitrokey/issues/140 to see if we
+    # can remove this Darwin-only branch.
+    hash = "sha256-4PEZ31QyVOmdhpKqTN8fwcHoLuu+w+OJ3fZeqwlE+io=";
+    fetchSubmodules = true;
```

Comment:
Could we set `fetchSubmodules` everywhere? Having multiple hashes could break automatic updates.

Reply:
Very good point, updated.